acceptance/localcluster/docker.go

```diff
@@ -130,7 +131,8 @@ func createContainer(client dockerclient.Client, config dockerclient.ContainerCo
 		return nil, err
 	}
 	return &Container{
-		ContainerInfo: dockerclient.ContainerInfo{Id: id},
+		ID:            id,
```

Comment:
The ContainerInfo is overwritten en masse in Container.Inspect whenever anything changes, while `Container.Id` (really `Container.ContainerInfo.Id`) was being accessed from other threads. Since the ID is accessed everywhere it's better to hoist it out into a separate constant field. I had to rename it from Id to ID to make golint happy.